src/app/app.component.html

```diff
@@ -19,7 +19,7 @@
                 </li>
                 <li class="nav-item">
                     <a class="nav-item nav-link" routerLink="/features">
-                        <i class="fa fa-star"></i>Features
+                        <i class="fa fa-star"></i>&nbsp;Features
```

Comment:
Can we use css padding here?

Reply:
Done. Updated all &nbsp's with css padding